rpm/docker-build.sh

```diff
@@ -14,4 +14,4 @@ echo
 echo "RPMs written to: $PWD/output/x86_64/"
 ls $PWD/output/x86_64/
 echo
-echo "Yum repodata written to: $PWD/output/x86_64/yum/"
+echo "Yum repodata written to: $PWD/output/x86_64/repodata/"
```

Comment:
Missed the incorrect "yum" there in my previous comment, this should just be: echo "Yum repo written to: $PWD/output/x86_64/", intent here was to output the location you could rsync for your entire repo, and the rpms live one level above repodata.